code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -241,7 +244,7 @@
 	use_power = 1
```

Comment:
not sure if this is a bool, but theres a few bools on this object that dont use defines

Reply:
No, this is not exactly a boolean. `use_power` doesn't use defines, though you are right that it should.